app/javascript/src/NewService/components/FormTemplate.jsx

```diff
@@ -0,0 +1,103 @@
+// @flow
+
+import React from 'react'
+import {CSRFToken} from 'utilities/utils'
+import type {ServiceDiscoveryFormTemplateProps, FormTemplateProps} from 'NewService/types'
+
+const getServiceDiscoveryProps = (): ServiceDiscoveryFormTemplateProps => {
```

Comment:
I need to go to `NewService/types` to see what these props are... which I believe are only used here.

Reply:
Also, why this can't be just an object declaration?

Reply:
Hmm, I think this is a remain from a previous commit, before refactoring...